save.py

```diff
@@ -195,6 +196,24 @@ def rare_chance_as_str(self) -> tuple[str, str, str]:
     def upcoming_boss(self) -> str:
         return self._data["boss"]
 
+    @property
+    def bottled_cards(self) -> str:
+        cards = []
+        if self._data["bottled_flame"]:
```

Comment:
I don't actually know if this checks existence of the json key. The "bottled_x" keys only exist once you pick up the relic.

Reply:
It does not. You want `if self._data("bottled_flame")`